src/elements/kw-share-card/kw-share-card.html

```diff
@@ -79,7 +79,11 @@
             }
             
             .remix-button {
-                @apply --kano-icon-button;
+                @apply --kano-button;
+                background: var(--color-orange);
+                right: 5px;
```

Comment:
I think the 'position' css property here is 'static', so 'right' and 'bottom' will not have any effect